### DISCUSSION TITLE
Search option in Docs

### DISCUSSION THREAD

**Discussion by Sukriti-sood** [remix-run/react-router#9846] [Proposals]
### What is the new or updated feature that you are suggesting?

Integrating Documentation Search option.  (Algolia, meilisearch or anything)

### Why should this feature be included?

Every popular package is Including Search option their documentation site and also integrating search enhance user experience

**Comment by sshquack** [comment]
This would be nice to have. Current workaround is to use site specific search `https://www.google.com/search?q=site%3Areactrouter.com+navigate` ([Example link](https://www.google.com/search?q=site%3Areactrouter.com+navigate))

**Reply by paul-vd** [reply]
Thanks for that tip, I also use the direct repository to find things, but as you can expect, there is an overflow of information:  https://github.com/search?q=repo%3Aremix-run%2Freact-router&type=code

maybe 2023 will be the year when we get search 👶

**Comment by orangeswim** [comment]
Literally am struggling to use react-router because I can't search the docs for examples or keywords....

**Comment by silvenon** [comment]
I really really want this too. 🙏 Sorry for my "+1" comment, but I strongly believe this would immensely improve React Router usage, and maybe decrease number of questions around here. 😅 Especially useful because many people are still migrating over to v6.

**Comment by brophdawg11** [comment]
I'm going to convert this to a discussion so it can go through our new [Open Development](https://remix.run/blog/open-development) process.  Please upvote the new Proposal if you'd like to see this considered!

**Comment by Reliquat** [comment]
Please add this (essential) feature. I'm using Google + Stackoverflow over the official documentation because it's very tedious to find answers.

**Comment by lukaw3d** [comment]
I made this for myself https://lukaw3d.github.io/react-router-v6-searchable-docs/

**Comment by JorensM** [comment]
I see that this issue has been open for 2 years now. I can work on this if maintainers approve.

**Comment by brookslybrand** [comment]
I am sincerely sorry this took so long to get search in our docs. I just added it, closing [the issue on the react-router-website repo](https://github.com/remix-run/react-router-website/issues/75)
